cpp/tests/test_abm_world.cpp

```diff
@@ -351,6 +353,8 @@ TEST(TestWorld, evolveMigration)
 
```

Comment:
Yes, but I don't think this should be it's own unit test. Instead we should run the normal unittests (or a subset) with OMP_NUM_THREADS > 1. This requires a bit of setup, since normal github runners don't really support multithreading.

Reply:
Created an issue for multi threaded tests: #786